tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -20,61 +20,63 @@ export const feature_mcolor3: Feature<string> = {
 
 export const flavor_text: Feature<string> = {
   name: 'Flavor Text',
-  description: 'Describe your character!',
+  description:
+    "Appears when your character is examined (but only if they're identifiable - try a gas mask).",
   component: FeatureTextInput,
 };
 
 export const silicon_flavor_text: Feature<string> = {
   name: 'Flavor Text (Silicon)',
-  description: 'Describe your cyborg/AI shell!',
+  description: "Only appears if you're playing as a borg/AI.",
   component: FeatureTextInput,
 };
 
 export const ooc_notes: Feature<string> = {
   name: 'OOC Notes',
-  description: 'Talk about your character OOCly!',
+  description:
+    'Primarily for things like ERP, general, antag/whatever preferences.',
```

Comment:
ERP visibility is a server preference, please rewrite this to not explicitly refer to it.

Reply:
we already have precedent to reference ERP stuff in character preference in the form of the exploitables default text

Reply:
We have preference to not reference ERP stuff by the fact we have an entire config that removes it.

Someone missing a config argument is not an argument to have it non-optional elsewhere